toolchains/gcc_compatibles.cmake

```diff
@@ -29,7 +29,7 @@ add_compile_options( -fstrict-aliasing -fstrict-enums -fvisibility=hidden -fvisi
 # "Unknown language" error with CMake 3.5.2 if COMPILE_LANGUAGE:C is used.
 # + 'COMPILE_LANGUAGE' isn't supported by VS generators:
 # https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html#logical-expressions
-add_compile_options( $<$<COMPILE_LANGUAGE:CXX>:-std=gnu++1z> )
+add_compile_options( $<$<COMPILE_LANGUAGE:CXX>:-std=gnu++14> )
```

Comment:
Qt 4.7 and PCH compatibility. When linking with targets created by Qt framework, they automatically publish compiler flag `-std=gnu++11` or `-std=gnu++14` to your code. OTOH, when assembling flags for PCH, this flag may still not be published (e.g. in cases target for which PCH is added depends on target which is not yet defined but which ultimately depends on Qt - PCH then ends up with `-std=gnu++1z`, whilst code ends up with `-std=gnu++14`. AFAIK, this issue can only be solved by CMake adding truly native support for PCH (without hacks we currently use).